pages/index.js

```diff
@@ -1,7 +1,106 @@
-export default function HomePage() {
-	return (
-		<>
-			<h1>My Cosmetics Store</h1>
-		</>
-	);
+import styled from "styled-components";
+
+import { Btn } from "@/components/Button";
+
+import React, { useState } from "react";
+
+import myComponent from "@/components/myComponent1/myComponent.js";
```

Comment:
improve import statements
```suggestion
import React, { useState } from "react";
import styled from "styled-components";

import Button from "@/components/Button";
import Title from "@/components/Title";
```